config/validated_config_db_connector.py

```diff
@@ -0,0 +1,24 @@
+import jsonpatch
+
+from swsscommon.swsscommon import SonicV2Connector, ConfigDBConnector
+from generic_config_updater.generic_updater import GenericUpdater, ConfigFormat
+
+class ValidatedConfigDBConnector(ConfigDBConnector):
```

Comment:
>ConfigDBConnector [](http://example.com/codeflow?start=33&length=17)

The base class is not used in the implementation. Decorator pattern makes more sense.

Reply:
Changed to decorator pattern